keel-ec2-plugin/keel-ec2-plugin.gradle

```diff
@@ -4,7 +4,7 @@ apply from: "$rootDir/gradle/junit5.gradle"
 
 dependencies {
   implementation project(":keel-retrofit")
-  implementation project(":keel-plugin")
+  api project(":keel-plugin")
```

Comment:
Is this so something in `keel-nflx` can see it?

Reply:
Yep - so that custom validation applied in keel-ec2-netflix can have these dependencies

Reply:
more pedantic than anything but I tend to sort the dependencies by scope (api, implementation, testImplementation) then the dependencies within a scope by project then alphabetically

if that sounds reasonable then just bumping this to the start of the list would be appreciated!

Reply:
I will make an attempt to sort things alphabetically in the future and fix here :)